scripts/remove-outdated-translations.js

```diff
@@ -0,0 +1,67 @@
+const fs = require("fs");
+const glob = require("glob");
+const path = require("path");
+
+const keyToUpdate = process.argv[2]; // Get the key to update from command line argument
+
+if (!keyToUpdate) {
+  console.error("Please provide the key to update as a command line argument.");
+  process.exit(1);
+}
+
+function updateNestedKey(obj, nestedKey) {
+  const keys = nestedKey.split(".");
+  let currentObj = obj;
+  let foundAndUpdated = false;
+
+  for (let i = 0; i < keys.length; i++) {
+    const key = keys[i];
+
+    if (i === keys.length - 1 && currentObj.hasOwnProperty(key)) {
+      currentObj[key] = "";
```

Comment:
Check if this can be removed completely?

Reply:
I now changed it to remove the key instead of resetting it to an empty string. Let's see if that makes a difference for weblate, I don't think so tbh.